neuron/neuron/contest.py

```diff
@@ -171,6 +171,14 @@ def __init__(self, message: str):
 ]
 
 
+def get_contest(contest_id: str) -> Contest:
```

Comment:
This is not needed if `find_contest` is a thing. You can do `ContestId[contest_id]` to get the ID from the name(if it's valid).